corehq/apps/hqwebapp/static/hqwebapp/js/base_ace.js

```diff
@@ -4,8 +4,7 @@ hqDefine('hqwebapp/js/base_ace', [
     'ace-builds/src-min-noconflict/mode-json',
 ], function (
     $,
-    ace,
-    jsonMode
```

Comment:
You could also slap on an [`// eslint-disable-line rulename`](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments) if you think this makes sense to keep